packages/nameguard-sdk/src/index.ts

```diff
@@ -448,6 +475,8 @@ const DEFAULT_NETWORK: Network = "mainnet";
 const DEFAULT_INSPECT_LABELHASH_PARENT = ETH_TLD;
 export const DEFAULT_COMPUTE_NAMEGUARD_REPORT = false;
 const MAX_BULK_INSPECTION_NAMES = 250;
+export const MAX_INSPECTED_NAME_CHARACTERS = 200;  // includes label separators
+const MAX_INSPECTED_NAME_UNKNOWN_LABELS = 5;
```

Comment:
@djstrong can we make a little comment here how even duplicated instances of the same unknown label are counted each time they appear in a name